fastlane/swift/LaneFileProtocol.swift

```diff
@@ -92,16 +92,25 @@ public class LaneFile: NSObject, LaneFileProtocol {
         }
     }
 
-    public static func runLane(named: String, parameters: [String: String]) -> Bool {
+    public static func runLane(from fastfile: LaneFile?, named: String, parameters: [String: String]) -> Bool {
         log(message: "Running lane: \(named)")
-        loadFastfile()
-
-        guard let fastfileInstance: Fastfile = self.fastfileInstance else {
-            let message = "Unable to instantiate class named: \(className())"
-            log(message: message)
-            fatalError(message)
-        }
-
+        #if !SWIFT_PACKAGE
+            loadFastfile()
+        #endif
```

Comment:
Can we add some comments here too on why this isn’t needed on Swift package version? 😇 I just want to know incase I need to come here and debug in the future ❤️